src/macaron/provenance/provenance_extractor.py

```diff
@@ -318,18 +304,6 @@ def check_if_input_purl_provenance_conflict(
             )
             return True
 
-    # Check the PURL commit against the provenance.
```

Comment:
Could you please remind me why we removed this commit validation ? 🤔

Reply:
Ah nevermind, I think I remembered it now. This is because for some projects the commit hash recorded in their provenances are different from the tagged commit. 
This is due to how their pipelines are setup, where the provenances recorded the commits that the pipelines were triggered on. The version tagged commits can be different if they bumped the versions during the pipeline.